scripts/manageacl.py

```diff
@@ -0,0 +1,158 @@
+import argparse
+import asyncio
+from enum import Enum
+
+from azure.core.credentials import AzureKeyCredential
+from azure.core.credentials_async import AsyncTokenCredential
+from azure.identity import AzureDeveloperCliCredential
+from azure.search.documents.aio import SearchClient
+from azure.search.documents.indexes.aio import SearchIndexClient
+from azure.search.documents.indexes.models import SearchFieldDataType, SimpleField
+
+class ManageAcl:
```

Comment:
Can you add module or class docstrings to explain what this does at a high-level?